src/api-reference/travel/itinerary-tmc-thirdparty/index.markdown

```diff
@@ -2508,6 +2512,7 @@ The booking elements contain many child elements. For ease of use, these element
 
 [1]: https://www.iata.org/publications/Pages/code-search.aspx
 [2]: https://en.wikipedia.org/wiki/ISO_4217
+[3]: https://developer.concur.com/api-reference/travel/itinerary/itinerary.html#semantics_codes
```

Comment:
This isn't a huge deal, but whenever possible links to our own site should not include the base URL. Reasoning is that we have a preview site (https://preview.developer.concur.com). Leaving off the base URL allows users on the preview site to be redirected appropriately. Ex: `/api-reference/travel/itinerary/itinerary.html#semantics_codes`.